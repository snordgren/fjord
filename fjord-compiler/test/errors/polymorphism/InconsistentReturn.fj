module errors.parametricPolymorphism.InconsistentReturn

import TestUtil

id : a. a -> a
id x =
  rm x 5
