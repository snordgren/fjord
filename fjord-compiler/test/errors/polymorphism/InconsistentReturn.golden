test/errors/polymorphism/InconsistentReturn.fj:5:3:
  |
5 |   5
  |   ^^
expression has type Int, expected a
